h/static/styles/variables.scss

```diff
@@ -42,6 +42,21 @@ $success-color: #1cbd41 !default;
 $mask-start-color: rgba($white, 0) !default;
 $mask-end-color: $white !default;
 
+
+// Variables for the new color palette
```

Comment:
There are still a bunch of constants at the bottom of this file which are also labeled as the "new color palette" - `$dg1 .. $lg4, $primary, $highlight`. Can you remove all of those which are now unused.